pages/services/index.vue

```diff
@@ -19,6 +19,7 @@
           format="webp"
           src="/img/hero-images/services.png"
           width="1344"
+          alt=""
```

Comment:
@hannakim91 do you plan to come back through and fill these in later? I think we should definitely make our alt tags descriptive.

Reply:
@rwwagner90 The w3 docs say to leave a null alt tag when it's a decorative image: I feel like the hero images are more-so ambiance/eye-candy vs things that would need to be read for context by screen readers?https://www.w3.org/WAI/tutorials/images/decorative/#image-used-for-ambiance-eye-candy

Reply:
Also thinking another run-through with manual tests like being able to tab through the pages w/o getting stuck in a section would be good - making a to-do list of those.

Reply:
> The w3 docs say to leave a null alt tag when it's a decorative image: I feel like the hero images are more-so ambiance/eye-candy vs things that would need to be read for context by screen readers?https://www.w3.org/WAI/tutorials/images/decorative/

I'm not sure I agree with that. Aren't all images decorative in that sense? I tend to provide descriptive tags for all of them. For example `alt="Ship Shape octopus mascot drinking a coffee and using a laptop."`. My thought was for every image a sighted person can see, there should be meaningful content for those using screen readers, but I could definitely be wrong on my understanding of that if those descriptions are actually annoying to people with screen readers.

Reply:
That was my understanding of it - like the alt text for the top hero image would be something like `Ship Shape octopus mascot holds a sign in an ocean filled with browser windows` doesn't really provide additional context for users with vision issues, and may be more confusing in the context of the header/CTA button?

In comparison to these (informative image examples)[https://www.w3.org/WAI/tutorials/images/informative/] - which have a reason for users to know what the images contain.

(This person)[https://www.portent.com/blog/design-dev/decorative-images-how-to-identify-and-tag-them-for-accessibility.htm] talks about similar use cases with stock images for "flavoring" blog posts too.